charts/vllm/examples/readmanypvc.yaml

```diff
@@ -0,0 +1,20 @@
+# This example requires first running
```

Comment:
Maybe rename this with a `-values.yaml` suffix

Reply:
fixed